Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: geopandas in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (1.0.1)
Requirement already satisfied: numpy>=1.22 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from geopandas) (2.2.5)
Requirement already satisfied: pyogrio>=0.7.2 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from geopandas) (0.11.0)
Requirement already satisfied: packaging in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from geopandas) (24.2)
Requirement already satisfied: pandas>=1.4.0 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from geopandas) (2.2.3)
Requirement already satisfied: pyproj>=3.3.0 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from geopandas) (3.7.1)
Requirement already satisfied: shapely>=2.0.0 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from geopandas) (2.1.0)
Requirement already satisfied: python-dateutil>=2.8.2 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from pandas>=1.4.0->geopandas) (2.9.0.post0)
Requirement already satisfied: pytz>=2020.1 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from pandas>=1.4.0->geopandas) (2025.2)
Requirement already satisfied: tzdata>=2022.7 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from pandas>=1.4.0->geopandas) (2025.2)
Requirement already satisfied: certifi in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from pyogrio>=0.7.2->geopandas) (2025.4.26)
Requirement already satisfied: six>=1.5 in c:\users\miguelsantos\appdata\roaming\python\python313\site-packages (from python-dateutil>=2.8.2->pandas>=1.4.0->geopandas) (1.17.0)
